doc/install.md

```diff
@@ -1,15 +1,35 @@
-### Building
+## Building and Installing Stacker
 
-Stacker requires at least go 1.11, since it uses go modules.
-On Ubuntu 18.04 you can get that using the instructions at:
+### Go Dependency
+
+Stacker requires at least go 1.11, since it uses Go modules.
```

Comment:
I'd just scratch ', since it uses Go modules'.

1.11 is 18 months old.

Reply:
That was just a carryover from the original file, I'll drop it.